tools/RAiDER/checkArgs.py

```diff
@@ -48,7 +48,7 @@ def checkArgs(args):
                     args.output_directory,
                     '{}_Delay_{}.csv'
                     .format(
-                        args.weather_model.Model(),
```

Comment:
Is this the same thing?

Reply:
For most weather models yes, but the former was 'ERA-5', the latter 'ERA5'